### PR TITLE
Adjusted responsive display

### PR DESCRIPTION
**Pull request by magmat88**
Adjusted responsive display - mobile first.
Applied best practices to SASS styles.

### CODE REVIEW

File: src/app/components/aboutPage/aboutPage.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@import '../../styles/mixins/mixins.scss';
```
Comment (earendillo) [inline src/app/components/aboutPage/aboutPage.component.scss:1]: no need of importing styles if they are not use within the file.

Comment (earendillo) [reply src/app/components/aboutPage/aboutPage.component.scss:1]: applies to all occurences

File: src/app/components/header/header.component.scss [hunk 2]
```diff
@@ -1,40 +1,69 @@
...
+}
+
+ul {
+    @extend %paddingZero;
+    list-style-type: none;
+}
+
+li {
+    @extend %paddingStandard;
+    border: 1px solid white;
```
Comment (earendillo) [inline src/app/components/header/header.component.scss:52]: use variable for color; do not use native css color names

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/components/aboutPage/aboutPage.component.scss (hunk 1 above)
Comment (magmat88) [reply src/app/components/aboutPage/aboutPage.component.scss:1]: removed all unused style imports

File: src/app/components/header/header.component.scss (hunk 2 above)
Comment (magmat88) [reply src/app/components/header/header.component.scss:52]: changed

**Pull request merged**